internal/backends/linuxkms/renderer/sw.rs

```diff
@@ -61,6 +61,10 @@ const SOFTWARE_RENDER_SUPPORTED_DRM_FOURCC_FORMATS: &[drm::buffer::DrmFourcc] =
 #[derive(Copy, Clone, bytemuck::Pod, bytemuck::Zeroable)]
 struct DumbBufferPixelXrgb888(pub u32);
 
+#[repr(transparent)]
+#[derive(Copy, Clone, bytemuck::Pod, bytemuck::Zeroable)]
+struct DumbBufferPixelBgra888(pub u32);
```

Comment:
Since we're now using Bgra8888 instead of Ba24, I think this type should also be called Bgra8888 instead of Bgra888. What do you think?

Reply:
Eeehr, I thought I replaced all occurrences of `BA24` with `Bgra888`. To be sure i grepped with `BA24` again, no occurrences:

```
andreas@t480s: backends/linuxkms/renderer git master[$?>] via rs v1.90.0
> grep -i 'BA24' sw.rs                                                                                                                                                                        
andreas@t480s: backends/linuxkms/renderer git master[$?>] via rs v1.90.0
```

Did I miss something?

Reply:
I meant `Bgra8888` instead of `Bgra888` - note the pending `8` :)

Reply:
Oh since there were others using three eights I thought this was intended and followed the pattern. Should I fix the others,too?

Reply:
The other one is `Xrgb`, so it's visually it's only three channels :). But with Bgra8888 the alpha matter, so I think it's effectively four channels of 8 bits each.